### ISSUE TITLE
VCD 10.2 and CSE 3.0.1 OVF upload to catalog fails

### ISSUE THREAD

**Issue by mkriegs** [vmware/container-service-extension#869]
Hey ,

i think its more reletatet to vcd-cli problem than cse but i try to install cse thats why i post over here.
It fails during the Upload of ova. I try it manuel and it fail also. I found out there are some problems behinde LB #https://github.com/vmware/vcd-cli/issues/249 , so i ty both over lb or direct to cell(cse server right next to vcd same network without nat) both fail.

can someone tell me what i doing wrong ?
best regrads

Michael Kriegs

```
root@cse:~# pip3 install --user vcd-cli --upgrade

Requirement already satisfied: vcd-cli in ./.local/lib/python3.7/site-packages (24.0.0)

Requirement already satisfied: keyring<=12.0.0,>=10.6.0 in ./.local/lib/python3.7/site-packages (from vcd-cli) (12.0.0)

Requirement already satisfied: tabulate>=0.7.5 in ./.local/lib/python3.7/site-packages (from vcd-cli) (0.8.7)

Requirement already satisfied: pycryptodome==3.6.6 in ./.local/lib/python3.7/site-packages (from vcd-cli) (3.6.6)

Requirement already satisfied: unittest-xml-reporting>=2.2.1 in ./.local/lib/python3.7/site-packages (from vcd-cli) (3.0.4)

Requirement already satisfied: colorama>=0.3.9 in ./.local/lib/python3.7/site-packages (from vcd-cli) (0.4.4)

Requirement already satisfied: pyvcloud<24.0.0,>=23.0.0 in ./.local/lib/python3.7/site-packages (from vcd-cli) (23.0.0)

Requirement already satisfied: click>=7.0 in ./.local/lib/python3.7/site-packages (from vcd-cli) (7.1.2)

Requirement already satisfied: entrypoints in /usr/lib/python3/dist-packages (from keyring<=12.0.0,>=10.6.0->vcd-cli) (0.3)

Requirement already satisfied: secretstorage in /usr/lib/python3/dist-packages (from keyring<=12.0.0,>=10.6.0->vcd-cli) (2.3.1)

Requirement already satisfied: python-dateutil>=2.8.1 in ./.local/lib/python3.7/site-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (2.8.1)

Requirement already satisfied: pygments>=2.2.0 in ./.local/lib/python3.7/site-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (2.7.1)

Requirement already satisfied: requests>=2.18.4 in /usr/lib/python3/dist-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (2.21.0)

Requirement already satisfied: PyYAML>=4.2b1 in ./.local/lib/python3.7/site-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (5.3.1)

Requirement already satisfied: humanfriendly>=4.8 in /usr/local/lib/python3.7/dist-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (4.18)

Requirement already satisfied: vcd-api-schemas-type>=9.1.2.dev10 in ./.local/lib/python3.7/site-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (9.1.2.dev10)

Requirement already satisfied: lxml>=4.2.1 in ./.local/lib/python3.7/site-packages (from pyvcloud<24.0.0,>=23.0.0->vcd-cli) (4.5.2)

Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.8.1->pyvcloud<24.0.0,>=23.0.0->vcd-cli) (1.12.0)

 

root@cse:~# cat .cse-logs/cse-install_2021-01-08_00-56-05.log

21-01-08 00:56:13 | remote_template_manager:104 - get_remote_template_cookbook | DEBUG :: Downloaded remote template cookbook from http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/template.yaml

21-01-08 00:56:13 | configure_cse:329 - install_cse | INFO :: Installing CSE on vCloud Director using config file 'encrypted-config.yaml'

21-01-08 00:56:13 | configure_cse:366 - install_cse | INFO :: Connected to vCD as system administrator: vcloud.business.pyur.com:443

21-01-08 00:56:13 | configure_cse:372 - install_cse | ERROR :: MQTT extension found. Use `cse upgrade` instead of 'cse install'.

21-01-08 00:56:13 | configure_cse:466 - install_cse | ERROR :: CSE Installation Error

Traceback (most recent call last):

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/configure_cse.py", line 373, in install_cse

    raise Exception(ext_found_msg)

Exception: MQTT extension found. Use `cse upgrade` instead of 'cse install'.

root@cse:~# cat .cse-logs/cse-install_2021-01-08_00-5

cse-install_2021-01-08_00-50-37.log  cse-install_2021-01-08_00-56-05.log  cse-install_2021-01-08_00-56-30.log  cse-install_2021-01-08_00-58-32.log 

root@cse:~# cat .cse-logs/cse-install_2021-01-08_00-50-37.log

21-01-08 00:50:45 | remote_template_manager:104 - get_remote_template_cookbook | DEBUG :: Downloaded remote template cookbook from http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/template.yaml

21-01-08 00:50:45 | configure_cse:329 - install_cse | INFO :: Installing CSE on vCloud Director using config file 'encrypted-config.yaml'

21-01-08 00:50:45 | configure_cse:366 - install_cse | INFO :: Connected to vCD as system administrator: vcloud.business.pyur.com:443

21-01-08 00:50:46 | configure_cse:543 - _register_cse_as_mqtt_extension | INFO :: MQTT extension is ready

21-01-08 00:50:46 | configure_cse:751 - _register_def_schema | INFO :: Registering defined entity schema

21-01-08 00:50:47 | configure_cse:809 - _register_def_schema | INFO :: Successfully registered defined entity type

Successfully added ReadWrite ACL for native defined entity to System Org

21-01-08 00:50:47 | configure_cse:922 - _setup_placement_policies | INFO :: Setting up placement policies for cluster types

21-01-08 00:50:48 | configure_cse:938 - _setup_placement_policies | INFO :: Creating global PVDC compute policy

21-01-08 00:50:48 | configure_cse:960 - _setup_placement_policies | INFO :: Creating placement policy 'native'

21-01-08 00:50:48 | pyvcloud_utils:392 - create_and_share_catalog | INFO :: Creating catalog 'PYUR-CSE'

21-01-08 00:50:49 | pyvcloud_utils:398 - create_and_share_catalog | INFO :: Created catalog 'PYUR-CSE'

21-01-08 00:50:49 | remote_template_manager:104 - get_remote_template_cookbook | DEBUG :: Downloaded remote template cookbook from http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/template.yaml

21-01-08 00:50:49 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cust.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cust.sh'...

21-01-08 00:50:50 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:50 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/init.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/init.sh'...

21-01-08 00:50:50 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:50 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/nfsd.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/nfsd.sh'...

21-01-08 00:50:50 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:50 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/mstr.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/mstr.sh'...

21-01-08 00:50:50 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:50 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/node.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/node.sh'...

21-01-08 00:50:50 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:50 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/docker-upgrade.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/docker-upgrade.sh'...

21-01-08 00:50:51 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:51 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/master-cni-apply.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/master-cni-apply.sh'...

21-01-08 00:50:51 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:51 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/master-k8s-upgrade.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/master-k8s-upgrade.sh'...

21-01-08 00:50:51 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:51 | utils:334 - download_file | INFO :: Downloading file from 'http://raw.githubusercontent.com/vmware/container-service-extension-templates/master/scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/worker-k8s-upgrade.sh' to '/root/.cse_scripts/ubuntu-16.04_k8-1.19_weave-2.6.5_rev1/cluster-upgrade/worker-k8s-upgrade.sh'...

21-01-08 00:50:51 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:50:52 | utils:334 - download_file | INFO :: Downloading file from 'https://cloud-images.ubuntu.com/releases/xenial/release-20180418/ubuntu-16.04-server-cloudimg-amd64.ova' to 'cse_cache/ubuntu-16.04-server-cloudimg-amd64.ova'...

21-01-08 00:53:27 | utils:343 - download_file | INFO :: Download complete

21-01-08 00:53:27 | pyvcloud_utils:310 - upload_ova_to_catalog | INFO :: Uploading 'ubuntu-16.04-server-cloudimg-amd64.ova' to catalog 'PYUR-CSE'

21-01-08 00:53:49 | configure_cse:466 - install_cse | ERROR :: CSE Installation Error

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 600, in urlopen

    chunked=chunked)

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 354, in _make_request

    conn.request(method, url, **httplib_request_kw)

  File "/usr/lib/python3.7/http/client.py", line 1244, in request

    self._send_request(method, url, body, headers, encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1290, in _send_request

    self.endheaders(body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1239, in endheaders

    self._send_output(message_body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1065, in _send_output

    self.send(chunk)

  File "/usr/lib/python3.7/http/client.py", line 987, in send

    self.sock.sendall(data)

  File "/usr/lib/python3.7/ssl.py", line 1015, in sendall

    v = self.send(byte_view[count:])

  File "/usr/lib/python3.7/ssl.py", line 984, in send

    return self._sslobj.write(data)

BrokenPipeError: [Errno 32] Broken pipe

 

During handling of the above exception, another exception occurred:

 

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 449, in send

    timeout=timeout

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 638, in urlopen

    _stacktrace=sys.exc_info()[2])

  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 367, in increment

    raise six.reraise(type(error), error, _stacktrace)

  File "/usr/lib/python3/dist-packages/six.py", line 692, in reraise

    raise value.with_traceback(tb)

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 600, in urlopen

    chunked=chunked)

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 354, in _make_request

    conn.request(method, url, **httplib_request_kw)

  File "/usr/lib/python3.7/http/client.py", line 1244, in request

    self._send_request(method, url, body, headers, encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1290, in _send_request

    self.endheaders(body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1239, in endheaders

    self._send_output(message_body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1065, in _send_output

    self.send(chunk)

  File "/usr/lib/python3.7/http/client.py", line 987, in send

    self.sock.sendall(data)

  File "/usr/lib/python3.7/ssl.py", line 1015, in sendall

    v = self.send(byte_view[count:])

  File "/usr/lib/python3.7/ssl.py", line 984, in send

    return self._sslobj.write(data)

urllib3.exceptions.ProtocolError: ('Connection aborted.', BrokenPipeError(32, 'Broken pipe'))

 

During handling of the above exception, another exception occurred:

 

Traceback (most recent call last):

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 695, in upload_ovf

    callback=callback)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 762, in _upload_file

    file_name, target_uri, chunk_size=chunk_size, callback=callback)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 840, in _upload_part_file

    target_uri, data, range_str)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/client.py", line 1321, in upload_fragment

    verify=self._verify_ssl_certs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 593, in put

    return self.request('PUT', url, data=data, **kwargs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 533, in request

    resp = self.send(prep, **send_kwargs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 646, in send

    r = adapter.send(request, **kwargs)

  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 498, in send

    raise ConnectionError(err, request=request)

requests.exceptions.ConnectionError: ('Connection aborted.', BrokenPipeError(32, 'Broken pipe'))

 

During handling of the above exception, another exception occurred:

 

Traceback (most recent call last):

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/configure_cse.py", line 450, in install_cse

    msg_update_callback=msg_update_callback)

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/configure_cse.py", line 1062, in _install_all_templates

    msg_update_callback=msg_update_callback)

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/configure_cse.py", line 1264, in _install_single_template

    retain_temp_vapp=retain_temp_vapp)

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/template_builder.py", line 463, in build

    self._upload_source_ova()

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/template_builder.py", line 216, in _upload_source_ova

    msg_update_callback=self.msg_update_callback)

  File "/usr/local/lib/python3.7/dist-packages/container_service_extension/pyvcloud_utils.py", line 312, in upload_ova_to_catalog

    org.upload_ovf(catalog_name, filepath)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 699, in upload_ovf

    e.__traceback__)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 695, in upload_ovf

    callback=callback)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 762, in _upload_file

    file_name, target_uri, chunk_size=chunk_size, callback=callback)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 840, in _upload_part_file

    target_uri, data, range_str)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/client.py", line 1321, in upload_fragment

    verify=self._verify_ssl_certs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 593, in put

    return self.request('PUT', url, data=data, **kwargs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 533, in request

    resp = self.send(prep, **send_kwargs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 646, in send

    r = adapter.send(request, **kwargs)

  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 498, in send

    raise ConnectionError(err, request=request)

pyvcloud.vcd.exceptions.UploadException: Ovf upload failed

 

 

vdc-cli:

 

root@cse:~# vcd login vcloud01.hlkomm.intern PYUR test --no-verify-ssl-certs -w

Password:

test logged in, org: 'PYUR', vdc: 'PYUR-CATALOG'

root@cse:~# vcd catalog upload PYUR-CSE -i TEST /root/cse_cache/ubuntu-16.04-server-cloudimg-amd64.ova

upload 94,371,840 of 284,948,480 bytes, 33%Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 600, in urlopen

    chunked=chunked)

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 354, in _make_request

    conn.request(method, url, **httplib_request_kw)

  File "/usr/lib/python3.7/http/client.py", line 1244, in request

    self._send_request(method, url, body, headers, encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1290, in _send_request

    self.endheaders(body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1239, in endheaders

    self._send_output(message_body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1065, in _send_output

    self.send(chunk)

  File "/usr/lib/python3.7/http/client.py", line 987, in send

    self.sock.sendall(data)

  File "/usr/lib/python3.7/ssl.py", line 1015, in sendall

    v = self.send(byte_view[count:])

  File "/usr/lib/python3.7/ssl.py", line 984, in send

    return self._sslobj.write(data)

BrokenPipeError: [Errno 32] Broken pipe

 

During handling of the above exception, another exception occurred:

 

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 449, in send

    timeout=timeout

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 638, in urlopen

    _stacktrace=sys.exc_info()[2])

  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 367, in increment

    raise six.reraise(type(error), error, _stacktrace)

  File "/usr/lib/python3/dist-packages/six.py", line 692, in reraise

    raise value.with_traceback(tb)

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 600, in urlopen

    chunked=chunked)

  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 354, in _make_request

    conn.request(method, url, **httplib_request_kw)

  File "/usr/lib/python3.7/http/client.py", line 1244, in request

    self._send_request(method, url, body, headers, encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1290, in _send_request

    self.endheaders(body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1239, in endheaders

    self._send_output(message_body, encode_chunked=encode_chunked)

  File "/usr/lib/python3.7/http/client.py", line 1065, in _send_output

    self.send(chunk)

  File "/usr/lib/python3.7/http/client.py", line 987, in send

    self.sock.sendall(data)

  File "/usr/lib/python3.7/ssl.py", line 1015, in sendall

    v = self.send(byte_view[count:])

  File "/usr/lib/python3.7/ssl.py", line 984, in send

    return self._sslobj.write(data)

urllib3.exceptions.ProtocolError: ('Connection aborted.', BrokenPipeError(32, 'Broken pipe'))

 

During handling of the above exception, another exception occurred:

 

Traceback (most recent call last):

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 695, in upload_ovf

    callback=callback)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 762, in _upload_file

    file_name, target_uri, chunk_size=chunk_size, callback=callback)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/org.py", line 840, in _upload_part_file

    target_uri, data, range_str)

  File "/root/.local/lib/python3.7/site-packages/pyvcloud/vcd/client.py", line 1321, in upload_fragment

    verify=self._verify_ssl_certs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 593, in put

    return self.request('PUT', url, data=data, **kwargs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 533, in request

    resp = self.send(prep, **send_kwargs)

  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 646, in send

    r = adapter.send(request, **kwargs)

  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 498, in send

    raise ConnectionError(err, request=request)

requests.exceptions.ConnectionError: ('Connection aborted.', BrokenPipeError(32, 'Broken pipe'))

 

Usage: vcd catalog upload [OPTIONS] <catalog-name> <file-name>

Try 'vcd catalog upload -h' for help.

 

Error: Ovf upload failed
```

**Comment by rocknes** [comment]
Hi Michael,

The broken pipe error message generally means that the server is forcibly closing the connection. Why it is doing so is hard to say.

I would suggest that you can try to upload the ova file manually via vCD UI. Name the template as `ubuntu-16.04-server-cloudimg-amd64.ova`. If you are able to upload the OVA from UI, you can rerun `cse template install` via cli and it will skip uploading the ova and continue with template installation.

Regards
Aritra Sen

**Comment by mkriegs** [comment]
Hey Aritra,

i try it and failed gain on the next step :(

First: i saw some IO Error ( Check it with 3 Different Storagepolicys) so it should be ok?!
Sec: the template Fail with error below "curl: (6) Could not resolve host: download.docker.com", from where the template gets his DNS server? my Network in vdc provides one, see the VM screen (onther server in the same subnet, internet access possible)

i provide the all logs.
I hope u can give me some advise. not such easy deployment cse native cluster :/

best regards Michael


process [2, <Response [200]>, <Response [200]>] on vm 'vim.VirtualMachine:vm-203397' finished, exit code: 2

Result: [2, <Response [200]>, <Response [200]>]

stderr:

resize2fs 1.42.13 (17-May-2015)

The filesystem is already 2593019 (4k) blocks long.  Nothing to do!

 

W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial/InRelease  Temporary failure resolving 'archive.ubuntu.com'

W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-updates/InRelease  Temporary failure resolving 'archive.ubuntu.com'

W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-backports/InRelease  Temporary failure resolving 'archive.ubuntu.com'

W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/xenial-security/InRelease  Temporary failure resolving 'security.ubuntu.com'

W: Some index files failed to download. They have been ignored, or old ones used instead.

curl: (6) Could not resolve host: download.docker.com

gpg: no valid OpenPGP data found.

 

stdout:

net.ipv6.conf.all.disable_ipv6 = 1

net.ipv6.conf.default.disable_ipv6 = 1

net.ipv6.conf.lo.disable_ipv6 = 1

NOCHANGE: partition 1 is size 20744159. it cannot be grown

installing kubernetes

Err:1 http://security.ubuntu.com/ubuntu xenial-security InRelease

  Temporary failure resolving 'security.ubuntu.com'

Err:2 http://archive.ubuntu.com/ubuntu xenial InRelease

  Temporary failure resolving 'archive.ubuntu.com'

Err:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease

  Temporary failure resolving 'archive.ubuntu.com'

Err:4 http://archive.ubuntu.com/ubuntu xenial-backports InRelease

  Temporary failure resolving 'archive.ubuntu.com'

Reading package lists...

Reading package lists...

Building dependency tree...

Reading state information...

apt-transport-https is already the newest version (1.2.26).

ca-certificates is already the newest version (20170717~16.04.1).

curl is already the newest version (7.47.0-1ubuntu2.7).

software-properties-common is already the newest version (0.96.20.7).

The following package was automatically installed and is no longer required:

  grub-pc-bin

Use 'apt autoremove' to remove it.

0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

 



Failed VM customization. Please check logs.

Template Installation Error. Check CSE install logs
![IOERROR](https://user-images.githubusercontent.com/23615999/104475210-1c7cfd80-55bf-11eb-9bc0-73e1423aa56f.png)
![dns](https://user-images.githubusercontent.com/23615999/104475289-361e4500-55bf-11eb-8fd0-a6952626e627.png)

[cse-install_2021-01-13_11-50-58.log](https://github.com/vmware/container-service-extension/files/5809614/cse-install_2021-01-13_11-50-58.log)

**Comment by mkriegs** [comment]
Hey Update from my Side,
i passed somehow the template Installation. after lots of trys and errors i made it.

Now i stuck at deploy cluster from GUI, im not able to fetch my Networks.
I got Isolate Networks(vdc), routed Networks(vdc) and Datacenter Group Networks in the VDC.  All actions from gui end into fail.
Both from Systemadmin as same for ORG Admins.
![Networks](https://user-images.githubusercontent.com/23615999/104960994-bbe22c00-59d5-11eb-8c67-6498b3dd6915.png)
![K8Cluster](https://user-images.githubusercontent.com/23615999/104960997-be448600-59d5-11eb-92dd-e460f37bb9a5.png)

from cli i was able to deploy testcluster, changes also possible with edit .yaml
![K8Cluster](https://user-images.githubusercontent.com/23615999/104961119-fe0b6d80-59d5-11eb-9f04-3485939ee9f1.png)


my CSE Server made lots of Tokens in vcd, how do I find out which one is the right one?

i Hope to here somethink
best regards Michael

**Comment by ltimothy7** [comment]
Hi @mkriegs,

We are glad you were able to get template installation to work. 

Where are you seeing several tokens made, and are they all active? Also, are you using AMQP or MQTT?

Thank you

**Comment by mkriegs** [comment]
Hey @ltimothy7 
Can the cells run both at the same time ? amqp and mqtt?
I setup Applaunch and CSE with MQTT -> what i can see u change to this protokoll ( i know its not HA atm, but i cant support rappitmqcluster with LBs in front)
I setup 1 CSE Server with systemd , so one of them should be ok, the rest ist probley made by the issues during the installtion prozess. So how can i find out the right one? or should i delete all and restart CSE Server ?
![Token](https://user-images.githubusercontent.com/23615999/105184734-f95acc80-5b2f-11eb-8ff1-079aa17ac3e9.png)

The Plugin Issue will be attach in the 3.0.2 or comes with the next vcd release ?

best regards

**Comment by ltimothy7** [comment]
Hi @mkriegs,

Only one of the MQTT or AMQP extensions should be used. You can verify by checking your config.yaml file; if there is an "amqp" section, then you are using MQTT; otherwise if there is an "mqtt" section, then you are using MQTT. If both sections are present, MQTT is used.

The MQTT extension registers tokens, but it should only have 1 token active at a time. Thank you for pointing this out; we can see that we have a bug in this token maintenance and will fix it as soon as we can.

**Comment by ltimothy7** [comment]
@mkriegs 
Also, you can delete all of the extension tokens. The next time you run `cse run`, a new token will be created.

**Comment by mkriegs** [comment]
Hey @ltimothy7 

In my config i only use mqtt. 
But on the cell side i probley run both 😅 ampq not used anymore what i can think of.
I try my best to support you guys!

I try to delete all and rerun to next Update.

Is 10.2.1 allready supported? The gui Bug ist fix there.

Best regards

**Comment by ltimothy7** [comment]
Hi @mkriegs 

The mqtt config section means you are using MQTT then; thank you for checking.

VCD 10.2.1 is compatible with 3.0.1. Which gui bug are you referring to?

Thank you

**Comment by mkriegs** [comment]
Hey @ltimothy7 

yes i mean config section.

this bug #825 , but it should be solved from release nodes in 10.2.1 .

best regards

**Comment by ltimothy7** [comment]
Thank you for letting us know @mkriegs.
